src/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Squoosh",
   "short_name": "Squoosh",
-  "start_url": "/",
+  "start_url": "/?utm_medium=PWA&utm_source=launcher",
```

Comment:
We're currently using a dimension and some custom code to track how the app was launched, but Analytics has a built in way for doing this with `utm_medium` and `utm_source`.